config/read.go

```diff
@@ -119,6 +119,9 @@ func getDefaultConfig() *ServerConfig {
 	if dbSslKeyContents := os.Getenv("DB_SSLKEY_CONTENTS"); dbSslKeyContents != "" {
 		config.DbSslKeyContents = dbSslKeyContents
 	}
+	if dataDirectory := os.Getenv(("DB_DATA_DIRECTORY")); dataDirectory != "" {
```

Comment:
Looks like you got extra parentheses around `"DB_DATA_DIRECTORY"` that are not needed.

Reply:
Thanks: I think the Go mode in VS Code switched how it handles completion and it's still tripping me up.